util/testutil/integration/oci.go

```diff
@@ -181,6 +182,13 @@ func runBuildkitd(args []string, logs map[string]*bytes.Buffer, uid, gid int) (a
 	if err := os.Chown(tmpdir, uid, gid); err != nil {
 		return "", nil, err
 	}
+	if err := os.MkdirAll(filepath.Join(tmpdir, "tmp"), 0711); err != nil {
```

Comment:
is this for the `/proc/self/mountinfo` check in the defer?

Reply:
yes

Reply:
I'm not sure if I'm expected to see a failure by checking out only `util/testutil/integration` without any of the changes in the main logic, but FYI tests still pass in that case.

Reply:
yes, there was no mount leak atm. this is a precaution to check for leaks as well, not only for extra releases